### ISSUE TITLE
Aspect ratio not preserved when using fixed imageSize

### ISSUE THREAD

**Issue by AndrewBennet** [SvenTiigi/WhatsNewKit#41]
## WhatsNewKit Environment

- WhatsNewKit version: 1.3.3
- Xcode version: 11.5
- Swift version: 5
- macOS version running Xcode: 10.15.5
- Dependency manager (SPM, Carthage, CocoaPods, Manually): SPM (in Xcode)

## What did you do?

Configured a WhatsNew.Item with an image from the iOS 13 SFSymbol library.

## What did you expect to happen?

I expected the aspect ratio of the image to be preserved, especially as `WhatsNewViewController.ItemsView.ImageSize.fixed` is documented as `/// A fixed height by keeping the aspect ratio`. 

## What happened instead?

The image was stretched into a square.

--
Example code:
```
let whatsNewExample = WhatsNew(
    title: "An example of using WhatsNewKit with SFSymbols",
    items: [
        WhatsNew.Item(
            title: "Item 1",
            subtitle: "This has an SFSymbol as its image",
            image: {
                if #available(iOS 13.0, *) {
                    let config = UIImage.SymbolConfiguration(pointSize: 50)
                    return UIImage(systemName: "ant.fill", withConfiguration: config)
                } else {
                    return nil
                }
            }()
        ),
        WhatsNew.Item(
            title: "Item 2",
            subtitle: "This has an SFSymbol as its image",
            image: {
                if #available(iOS 13.0, *) {
                    let config = UIImage.SymbolConfiguration(pointSize: 50)
                    return UIImage(systemName: "tropicalstorm", withConfiguration: config)
                } else {
                    return nil
                }
            }()
        )
    ]
)

var configuration = WhatsNewViewController.Configuration()
// Use the default, preferred image size
//configuration.itemsView.imageSize = .original
window.rootViewController!.present(WhatsNewViewController(whatsNew: whatsNewExample, configuration: configuration), animated: true)
```

<img src="https://user-images.githubusercontent.com/2078225/84440787-80f36b00-ac32-11ea-9ecb-fa517f30d49f.png" width="400"/>

If I alter to code to set `.original` as the image size, the image is no longer distorted but I then do not have a nice way to ensure that the images are equal width to make them align, and so the text of each item don't have aligning leading margins:

<img src="https://user-images.githubusercontent.com/2078225/84440797-8650b580-ac32-11ea-99cc-2fbc322b6c5b.png" width="400" />

It would seem that the resizing code in WhatsNewItemsViewController+Cell.swift, line 207, `    func resize(height: CGFloat) -> UIImage?` doesn't do as its comment says: `/// Resize UIImage to a give height by keeping the aspect ratio`.

**Comment by AndrewBennet** [comment]
This seems like a great project though, thanks for making it! I'm hoping to add it in to my app [Reading List](https://github.com/AndrewBennet/ReadingList), but I'd much rather use iOS 13 SFSymbols than have to source my own images. I can use the original size setting, but the non-aligning text bugs me.

<img src="https://user-images.githubusercontent.com/2078225/84441778-47bbfa80-ac34-11ea-8e27-9aa74cc4d15e.png" width="400" />

**Comment by SvenTiigi** [comment]
Hey @AndrewBennet 

Thank you for your detailed bug report. 
I will investigate the reported bug and respond back if a fix is available ✌️

**Comment by AndrewBennet** [comment]
Hi Sven -- I've had a quite go at improving this, and submitted the change as a pull request.

Running the example app with the images swapped out for SFSymbol images, I see better results now: no distorted images, and no misaligned text.

<img src="https://user-images.githubusercontent.com/2078225/84827569-91358c80-b01c-11ea-9f9e-462174e4f87f.png" width="300" />

**Comment by SvenTiigi** [comment]
@AndrewBennet your PR (https://github.com/SvenTiigi/WhatsNewKit/pull/42) has been merged and a new release of WhatsNewKit is now available [1.3.4](https://github.com/SvenTiigi/WhatsNewKit/releases/tag/1.3.4)
